cpp/src/gandiva/precompiled/arithmetic_ops_test.cc

```diff
@@ -101,4 +101,29 @@ TEST(TestArithmeticOps, TestDiv) {
   context.Reset();
 }
 
+TEST(TestArithmeticOps, TestBitwiseOps) {
+  // bitwise AND
+  EXPECT_EQ(bitwise_and_int32_int32(5245, 94553), 4185);
```

Comment:
Using integer values for testing bitwise operations is hard to read and verify could you use hex values instead.

Reply:
Done.